README.md

```diff
@@ -14,10 +14,18 @@ authorization logic. You define this logic separately from the rest of your
 application code, but it executes inside the application and can call
 directly into it. oso ships as a library with a built-in debugger and REPL.
 
+oso is ideal for building permissions into user-facing applications, but you can
+check out [Use Cases](https://docs.oso.dev/more/use-cases.html) to learn about
+other applications for oso.
+
+Using oso consists of two parts:
+
+1. Writing oso policies in a declarative policy language called Polar
+2. Embedding oso in your application using the appropriate language-specific authorization library
```

Comment:
```suggestion
1. Writing oso policies in a declarative policy language called Polar.
2. Embedding oso in your application using the appropriate language-specific authorization library.
```

Reply:
Not sure we need the period since these aren't full sentences

Reply:
works for me